lib/tbot/testdata/TestSDS_FetchSecrets/all.golden

```diff
@@ -1,42 +1,42 @@
 {
-  "resources": [
+  "resources":  [
```

Comment:
Looks like this whitespace go added by running `go test` with `GOLDEN_UPDATE=1` - which is a little odd 🤔